components/config/opts.rs

```diff
@@ -886,7 +881,7 @@ pub fn from_cmdline_args(mut opts: Options, args: &[String]) -> ArgumentParsingR
             })
         });
 
-    let devtools_port = opt_match.opt_default("devtools", "6000").map(|port| {
+    let devtools_port = opt_match.opt_default("devtools", "0").map(|port| {
```

Comment:
Please add a comment mentioning that `0` mean random port.

Reply:
Done!